backend/api/namespace.go

```diff
@@ -127,6 +116,29 @@ func (a *NamespaceClient) ListNamespaces(ctx context.Context) ([]*corev2.Namespa
 					delete(namespaceMap, name)
 				}
 			}
+
+			return true
+		}
+
+		// Determine if this ClusterRoleBinding provides implicit access to
+		// namespaced resources
+		if binding.GetObjectMeta().Namespace == "" {
+			for _, resource := range rule.Resources {
+				if stringsutil.InArray(resource, corev2.CommonCoreResources) {
+					// All resources of type "namespace" are allowed
+					logger.Debugf("all namespaces implicitely authorized by the binding %s", binding.GetObjectMeta().Name)
```

Comment:
at the risk of being a pedant... `s/implicitely/implicitly/g`

Reply:
Good catch! 😅